README.textile

```diff
@@ -6,6 +6,9 @@ h1. url_validation -- Simple URL validator for Rails 3
 
 h2. About
 
+Note: This is Art19's fork of the url_validation gem.  Our fork contains the fix for validating
+URL's that have spaces and tabs in them.
+
```

Comment:
since this is our fork now, do we like to add our travis badge on top (and how about cc integration).